### ISSUE TITLE
FR: add SQLCHECK option to gixsql

### ISSUE THREAD

**Issue by GitMensch** [mridoni/gixsql#185]
See Oracle Pro*COB [docs "Syntactic and Semantic Checking"](https://docs.oracle.com/en/database/oracle/oracle-database/23/lnpcb/syntactic-and-semantic-checking.html#GUID-49010C30-E638-4ED4-9909-DC5F58C4D9D9) to prevent bad SQL (type mismatch) or non-existing tables/columns/typos to only happen at runtime.

Additional this may allow to provide an option to do parts of the currently dynamic parts happen already at compile time (setup of DB types/lengths/...) speeding up the actual access.

**Comment by GitMensch** [comment]
@lefessan that's the missing piece we discussed about for the migration you're looking into, right?

**Comment by lefessan** [comment]
Yes, but it's only an issue, not a pull-request, unless I misunderstood what you mean by "missing piece"
